modules/installation-user-infra-machines-iso.adoc

```diff
@@ -59,19 +59,23 @@ coreos.inst=yes
 coreos.inst.install_dev=sda <1>
 coreos.inst.image_url=<bare_metal_image_URL> <2>
 coreos.inst.ignition_url=http://example.com/config.ign <3>
-ip=<dhcp or static IP address> <4>
+ip=<dhcp or static IP address> <4> <5>
 ----
 <1> Specify the block device of the system to install to.
 <2> Specify the URL of the RAW image that you uploaded to your server.
 <3> Specify the URL of the Ignition config file for this machine type.
-<4> Set `ip=dhcp` or set an individual static IP address (ip=) and DNS server (nameserver=) on each node.
+<4> Set `ip=dhcp` or set an individual static IP address (`ip=`) and DNS server (`nameserver=`) on each node.
 For example, setting
 `ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none nameserver=4.4.4.41` sets:
-* Node's IP address to 10.10.10.2
-* Gateway address to 10.10.10.254
-* Netmask to 255.255.255.0
-* Hostname to core0.example.com
-* The DNS server address to 4.4.4.41
+* Node's IP address to `10.10.10.2`
+* Gateway address to `10.10.10.254`
+* Netmask to `255.255.255.0`
+* Hostname to `core0.example.com`
+* The DNS server address to `4.4.4.41`
+<5> You can apply the same link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/kickstart-and-advanced-boot-options_system-design-guide#network-boot-options_kickstart-and-advanced-boot-options[network boot options] to {op-system} as Red Hat Enterprise Linux 8. See . For example, if you use multiple network interfaces or DNS servers, you can further customize the IP address configuration in the following ways:
```

Comment:
That `See .` looks like it is dangling on its own and should be removed.

Reply:
Also, I'm now feeling kind of uneasy linking to those docs, since not all the network boot options listed in the RHEL docs are supported by RHCOS.  

The `ip=` and `nameserver=` options would be of most interest for RHCOS users.

Reply:
I've pulled the link. @miabbott, will you please take another look?